modules/user/client-react/signUp/actions/clearUser.ts

```diff
@@ -0,0 +1,7 @@
+import { ActionFunction, ActionType } from '.';
+
+const CLEAR_USER: ActionFunction<ActionType> = () => ({
```

Comment:
I have never seen functions to be called like this. Could you please give me a link to this convention.

Reply:
Done